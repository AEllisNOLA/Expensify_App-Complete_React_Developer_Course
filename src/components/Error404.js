import React from 'react';
import { Link } from 'react-router-dom';

const Error404 = () => (
    <div>
        <p>Error404 Component</p>
        <Link to='/'>Go to Dashboard</Link>
    </div>
)

export default Error404