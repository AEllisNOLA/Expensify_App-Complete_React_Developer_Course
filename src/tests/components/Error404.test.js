import React from 'react'
import { shallow } from 'enzyme'
import Error404 from '../../components/Error404'

test("Should properly render Error404 component", () => {
    const wrapper = shallow(<Error404 />)
    expect(wrapper).toMatchSnapshot()
})