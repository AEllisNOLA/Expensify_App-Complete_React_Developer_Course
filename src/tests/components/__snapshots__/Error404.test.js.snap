// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should properly render Error404 component 1`] = `
<div>
  <p>
    Error404 Component
  </p>
  <Link
    to="/"
  >
    Go to Dashboard
  </Link>
</div>
`;
